admin/release.py

```diff
@@ -506,3 +542,136 @@ def publish_rpms_main(args, base_path, top_level):
         raise SystemExit(1)
     finally:
         scratch_directory.remove()
+
+
+def create_release_branch(version, path):
```

Comment:
As a general design principle it would be better to separate the function that calculates the base branch from the function that creates the release branch, because it's good to separate read-only calculations that return something and side-effecty operations that change something. Eiffel calls these `functions` and `procedures` respectively. 

In practice I wouldn't bother changing it at this point, but worth keeping in mind for next time.